R/content.R

```diff
@@ -93,11 +93,25 @@ Content <- R6::R6Class(
     get_dashboard_url = function(pane = "") {
       dashboard_url_chr(self$connect$server, self$content$guid, pane = pane)
     },
-    #' @description Return the jobs for this content.
+    #' @description Return the jobs for this content
```

Comment:
Restoring `.` to match other `@description` fields that contain a single sentence.

```suggestion
    #' @description Return the jobs for this content.
```